doc/sphinx/README.template.rst

```diff
@@ -0,0 +1,93 @@
+=============================
+ Documenting Coq with Sphinx
+=============================
+
+..
+   This README is auto-generated from the coqrst docs; use ``./regen_readme.py`` to rebuild the it.
```

Comment:
Is the template really auto generated?

Reply:
No, of course not :) — but what simpler way was there to get the comment in the README? I've just pushed a clarification. It now says this:

> README.rst is auto-generated from README.template.rst and the coqrst docs;
> use ``doc/tools/coqrst/regen_readme.py`` to rebuild it.

Reply:
>what simpler way was there to get the comment in the README

Generate it in the python program? Your solution is fine too.